src/07-projects/projects.apib

```diff
@@ -111,6 +111,22 @@ Create a new project.
         + title: `New company website` (string, required)
         + description (string, optional)
         + starts_on: `2016-02-04` (string, required)
+        + billing_method (object, optional)
+            + One Of
+                + Properties
+                    + method: `fixed` (string, required)
+                    + amount (Money, required)
+                + Properties
+                    + method: `time_material` (string, required)
+                    + hourly_rate (object, required)
+                        + One Of
+                            + Properties
+                                + based_on: `user` (string, required)
+                            + Properties
+                                + based_on: `work_type` (string, required)
+                            + Properties
+                                + based_on: `custom` (string, required)
+                                + rate (Money, required)
```

Comment:
Q: Should we call this `amount` instead? Would be consistent with the naming when `method` is `fixed`